.github/workflows/shaptools-ci.yml

```diff
@@ -0,0 +1,103 @@
+name: Formula CI
```

Comment:
Change this `Formula CI` by `Package CI` for example

Reply:
Okay